src/test/logical_query_plan/lqp_find_subplan_mismatch_test.cpp

```diff
@@ -148,4 +148,15 @@ TEST_F(LQPFindSubplanMismatchTest, AdditionalNode) {
   EXPECT_EQ(mismatch->second, additional_predicate_node);
 }
 
+TEST_F(LQPFindSubplanMismatchTest, TypeMismatch) {
+  const auto first_node = PredicateNode::make(between(_query_nodes_rhs.table_a_a, 42, 45));
+  const auto second_node = PredicateNode::make(between(_query_nodes_rhs.table_a_a, 42ll, 45ll));
```

Comment:
I think we should use `int64_t{42}` and `int32_t{42}` if we want to make types explicit, imo.

Reply:
Good idea.